src/pages/pricing/index.tsx

```diff
@@ -223,8 +223,7 @@ export default function Pricing() {
                     ✓
                   </span>
                   <div>
-                    Student pays <br />
-                    <small>(coming Fall 2023)</small>
+                    Student pays
```

Comment:
The enclosing div is now technically unnecessary if you want to match the "course pays" button.